mobile/lib/infrastructure/entities/store.entity.dart

```diff
@@ -0,0 +1,12 @@
+import 'package:isar/isar.dart';
+
+part 'store.entity.g.dart';
+
+@Collection(accessor: 'store')
+class StoreEntity {
+  final Id id;
+  final int? intValue;
+  final String? strValue;
```

Comment:
Again, probably lack of dart knowledge, but (1) having the data type in the field name kind of sucks and (2) as I understand it either of them will always be undefined? If that's the case, wouldn't a generic make more sense? (or only storing the string and parsing it, idk)